packages/mdc-button/_mixins.scss

```diff
@@ -288,12 +309,17 @@
       pointer-events: none;
     }
   }
+
+  .mdc-button__label {
```

Comment:
Why is this & icon positioned relative?

Reply:
Removed. Originally this was added to make label/icon positioned above ripple in GM, but I'll add a separate GM mixin for that later.